server/tests/mochData/socialMockData.js

```diff
@@ -0,0 +1,41 @@
+const mockData = [
+  {
+    username: 'Patsy Waters',
+    firstname: 'Patsy',
+    lastname: 'Waters',
+    email: 'Anais79@hotmail.com',
+    role: 'Requester',
+    isVerified: true,
+    googleId: 'f02369b8-0081-4691-a732-516e59994234',
+    isGoogleSigned: true
+  },
+  {
+    id: '105602922558884032675',
+    username: 'Champs Dev',
+    firstname: 'Champs',
+    lastname: 'Dev',
+    email: 'champsdev2@gmail.com',
```

Comment:
This email is valid. Use an invalid email instead.

Reply:
Feedback Implemented! using fake email now